docs/integrations/sources/low-code.md

```diff
@@ -0,0 +1,10 @@
+# Low-code
+
+This doc is currently used for changelog purposes only internally, so that we can make sure that the source stays
+up-to-date with the latest version of the CDK. The doc is hidden from the docs site.
+
+## CHANGELOG
```

Comment:
Nit: let's also mention this file is automatically updated as part of the cdk publish flow

Reply:
done in [5f9f9d9](https://github.com/airbytehq/airbyte/pull/36501/commits/5f9f9d9694a5a687b0c406ff89df694e1de3612a)